api/server/router/build/backend.go

```diff
@@ -9,8 +9,7 @@ import (
 // Backend abstracts an image builder whose only purpose is to build an image referenced by an imageID.
 type Backend interface {
 	// Build a Docker image returning the id of the image
-	// TODO: make this return a reference instead of string
-	Build(context.Context, backend.BuildConfig) (string, error)
+	Build(context.Context, backend.BuildConfig) (*string, error)
```

Comment:
I guess this TODO was a bit ambiguous. I believe it should have read ` return a distribution.Reference`. It wasn't suggesting that it should return a `*string`